starter-code/index.html

```diff
@@ -37,6 +37,7 @@
           <div class="byline">
             By <address><a href="">Author Name</a></address>
             published <time pubdate datetime="2000-01-01">Publish Time</time>
+            <p></p>
```

Comment:
You should not be editing the given html unless the instructions specifically ask you to. Instead of creating a new p tag, wouldn't it be better to target a specific class? Maybe article-body!?